### ISSUE TITLE
Check for CA9

### ISSUE THREAD

**Issue by JennaMalkin** [Creatures-Developer-Network/C2toDS-Refresh#50]
Check that CA9 is not being used by flowers or plants or critters (to navigate to plants/flowers).

**Comment by bedalton** [comment]
CA9 is used in `HIRP` commands by Ants, Bats, Bees, Bluebirds, all catapillars, frogs and hummingbirds.

It is also used with `EMIT` commands by pears, death-cap mushrooms, Foxfire tree, pears, sunflowers, triffids and trumpets.

If this CA should not be used what should one use instead?

Should navigation happen through more direct means of selecting a target?

**Comment by Vermidia** [comment]
yeah, using esee or star would be better in this case. Tracking areas with high nutrients/moisture might also work, as plants would grow there.
